src/WebApi/Program.cs

```diff
@@ -1,3 +1,5 @@
+using HealthChecks.UI.Client;
```

Comment:
Can this using be removed?

`using HealthChecks.UI.Client;`

Reply:
No. It is required for the ResponseWriter of the health-report endpoint.